packages/cactus-plugin-ledger-connector-iroha2/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@hyperledger/cactus-plugin-ledger-connector-iroha2",
+  "version": "1.0.0",
+  "description": "Allows Cactus nodes to connect to an Iroha V2 ledger.",
+  "keywords": [
+    "Hyperledger",
+    "Cactus",
+    "Iroha",
+    "Iroha2",
+    "Iroha V2",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cactus.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cactus Contributors",
+    "email": "cactus@lists.hyperledger.org",
+    "url": "https://www.hyperledger.org/use/cactus"
+  },
+  "contributors": [
+    {
+      "name": "Michal Bajer",
+      "email": "michal.bajer@fujitsu.com",
+      "url": "https://www.fujitsu.com/global/"
+    }
+  ],
+  "main": "dist/lib/main/typescript/index.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/types/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "codegen": "run-p 'codegen:*'",
+    "codegen:openapi": "npm run generate-sdk",
+    "generate-sdk": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/ --reserved-words-mappings protected=protected"
+  },
+  "dependencies": {
+    "@hyperledger/cactus-common": "1.0.0",
```

Comment:
@outSH FYI: Once the 1.1.0 release goes out (which is pending ATM) you'll need to manually update these versions to 1.1.0 as well (otherwise the build will start breaking in really strange ways in some places). There's no automated way to do this yet for PRs that get caught up in a release, apologies for the complication!